changelogs/fragments/76735-ansible-galaxy-fix-ignore-certs.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - ansible-galaxy - Fix using the '--ignore-certs' option when there are no configured galaxy servers.
```

Fixing wording since this bug applies to both unconfigured servers_list and using a server via `-s` that isn't included in the configured servers_list.
```suggestion
  - ansible-galaxy - Fix using the '--ignore-certs' option when there is no server-specific configuration for the Galaxy server.
```